modules/oxide/src/less/theme/components/toolbar-button/toolbar-button.less

```
@@ -34,10 +35,12 @@
 
 @toolbar-button-active-background-color: @toolbar-button-enabled-background-color;
 @toolbar-button-active-border: @toolbar-button-enabled-border;
-@toolbar-button-active-box-shadow: @toolbar-button-box-shadow; // set to `0 1px 1px 0 rgba(0, 0, 0, .1) inset` for a small push in effect
+@toolbar-button-active-box-shadow: @toolbar-button-box-shadow;
+/* set to `0 1px 1px 0 rgba(0, 0, 0, .1) inset` for a small push in effect */
 @toolbar-button-active-icon-color: @toolbar-button-enabled-icon-color;
 @toolbar-button-active-text-color: @toolbar-button-enabled-text-color;
-@toolbar-button-active-transform: none; // Set to `translateY(1px)` for a small push down effect
+@toolbar-button-active-transform: none;
+/* Set to `translateY(1px)` for a small push down effect */
```

Comment moved down instead of up
```suggestion
/* Set to `translateY(1px)` for a small push down effect */
@toolbar-button-active-transform: none;
```